browser_extension/js/main.js

```diff
@@ -325,7 +418,13 @@ $(function () {
 
   // Whenever the server emits 'user joined', log it in the chat body
   socket.on('user joined', function (data) {
-    log(data.username + ' joined');
+
+    chrome.storage.local.get(['key'], function(result) {
```

Comment:
We can request a message from the background script that will return the authorization status of a Google account.
See the PR comments for more details.

We should also consider assigning that login state to a user object.